### PR TITLE
Add information about Rider support

### PR DESCRIPTION
**Pull request by grokys**
**What kind of change does this PR introduce?**

Rider now ships with inbuilt Avalonia support. Generalize stuff that was previously VS-specific to also include Rider.

### CODE REVIEW

File: src/AvaloniaUI.Net/wwwroot/docs/quickstart/ide-support.md [hunk 1]
```diff
@@ -1,8 +1,18 @@
 ---
-Title: Visual Studio Designer
+Title: IDE Support
 Order: 28
 ---
 
+## JetBrains Rider
+
+The [JetBrains Rider](https://www.jetbrains.com/rider/) IDE has inbuilt support for Avalonia XAML starting in 2020.2 including first-class support for Avalonia-specific XAML features and custom code inspections.
```
Comment (rstm-sf) [inline src/AvaloniaUI.Net/wwwroot/docs/quickstart/ide-support.md:8]: Isn't it from the 2020.3 version?

File: src/AvaloniaUI.Net/wwwroot/docs/index.md [hunk 2]
```diff
@@ -5,7 +5,9 @@ Order: 50
 
 Welcome to Avalonia! Avalonia is a cross platform XAML Framework for .NET Framework, .NET Core and Mono.
 
-Avalonia is shipped as .NET standard 2.0 [NuGet packages](/docs/quickstart/packages) and a [set of templates](/docs/quickstart/create-new-project) for Visual Studio and .NET core. We also have a [visual designer](/docs/quickstart/vs-designer) for Visual Studio.
+Avalonia is shipped as .NET standard 2.0 [NuGet packages](/docs/quickstart/packages) and a [set of templates](/docs/quickstart/create-new-project) for the .NET CLI.
+
+The [JetBrains Rider](https://www.jetbrains.com/rider/) IDE has inbuilt support for Avalonia XAML [starting in 2020.2](https://www.jetbrains.com/rider/whatsnew/#avalonia-support) including first-class support for Avalonia-specific XAML features and custom code inspections. We also have a [Visual Studio extension](/docs/quickstart/ide-support).
```
Comment (khalidabuhakmeh) [inline src/AvaloniaUI.Net/wwwroot/docs/index.md:10]:
```suggestion
The [JetBrains Rider](https://www.jetbrains.com/rider/) IDE has built-in support for Avalonia XAML [starting in 2020.3](https://www.jetbrains.com/rider/whatsnew/2020-3/#version-2020-3-avalonia-support) including first-class support for Avalonia-specific XAML features and custom code inspections. We also have a [Visual Studio extension](/docs/quickstart/ide-support).
```

Comment (khalidabuhakmeh) [reply src/AvaloniaUI.Net/wwwroot/docs/index.md:10]: I added an _evergreen_ link to make sure that the link doesn't get broken.

File: src/AvaloniaUI.Net/wwwroot/docs/quickstart/create-new-project.md [hunk 3]
```diff
@@ -2,6 +2,16 @@
 Title: Creating a new Avalonia Project
 Order: 0
 ---
+## JetBrains Rider
+
+The [JetBrains Rider](https://www.jetbrains.com/rider/) IDE has inbuilt support for Avalonia XAML [starting in 2020.2](https://www.jetbrains.com/rider/whatsnew/#avalonia-support) including first-class support for Avalonia-specific XAML features and custom code inspections.
```
Comment (khalidabuhakmeh) [inline src/AvaloniaUI.Net/wwwroot/docs/quickstart/create-new-project.md:7]:
```suggestion
The [JetBrains Rider](https://www.jetbrains.com/rider/) IDE has built-in support for Avalonia XAML [starting in 2020.3](https://www.jetbrains.com/rider/whatsnew/2020-3/#version-2020-3-avalonia-support) including first-class support for Avalonia-specific XAML features and custom code inspections.
```

File: src/AvaloniaUI.Net/wwwroot/docs/quickstart/ide-support.md (hunk 1 above)
Comment (khalidabuhakmeh) [inline src/AvaloniaUI.Net/wwwroot/docs/quickstart/ide-support.md:8]:
```suggestion
The [JetBrains Rider](https://www.jetbrains.com/rider/) IDE has built-in support for Avalonia XAML [starting in 2020.3](https://www.jetbrains.com/rider/whatsnew/2020-3/#version-2020-3-avalonia-support) including first-class support for Avalonia-specific XAML features and custom code inspections.
```

File: src/AvaloniaUI.Net/wwwroot/docs/quickstart/ide-support.md [hunk 4]
```diff
@@ -1,8 +1,18 @@
...
+
+See the [blog post](https://www.jetbrains.com/rider/whatsnew/#avalonia-support) for more information,.
```
Comment (khalidabuhakmeh) [inline src/AvaloniaUI.Net/wwwroot/docs/quickstart/ide-support.md:10]:
```suggestion
See the [JetBrains Rider 2020.3 release announcement](https://www.jetbrains.com/rider/whatsnew/2020-3/#version-2020-3-avalonia-support) for more information,.
```

**Review by khalidabuhakmeh — CHANGES_REQUESTED** [review]
I've added evergreen links in place of directing to the what's new page directly. This will make sure these links don't get broken as we release new versions of Rider.

File: src/AvaloniaUI.Net/wwwroot/docs/quickstart/ide-support.md [hunk 5]
```diff
@@ -1,8 +1,18 @@
...
+
+Rider does not yet provide a visual designer, but this is in development. See [the GitHub project](https://github.com/ForNeVeR/AvaloniaRider) for more information and instructions on how to install.
+
+## Visual Studio
+
 The [Avalonia for Visual Studio extension](https://marketplace.visualstudio.com/items?itemName=AvaloniaTeam.AvaloniaforVisualStudio) includes a XAML designer which can be used to show a live preview of the XAML as you're writing it. With the Avalonia for Visual Studio extension installed, double click on an Avalonia XAML file to open it.
```
Comment (khalidabuhakmeh) [inline src/AvaloniaUI.Net/wwwroot/docs/quickstart/ide-support.md:16]:
```suggestion
### Avalonia Visual Studio Extension

The [Avalonia for Visual Studio extension](https://marketplace.visualstudio.com/items?itemName=AvaloniaTeam.AvaloniaforVisualStudio) includes a XAML designer which can be used to show a live preview of the XAML as you're writing it. With the Avalonia for Visual Studio extension installed, double click on an Avalonia XAML file to open it.

### ReSharper for Visual Studio

For Visual Studio and ReSharper users, [ReSharper 2020.3 introduces](https://www.jetbrains.com/resharper/whatsnew/2020-3/#version-2020-3-avalonia-support) built-in code analysis, code completion, navigation, and find usages.
```

**Comment by grokys** [comment]
Thanks @khalidabuhakmeh, suggestions committed!

**Pull request merged**